src/Bicep.Core.IntegrationTests/ScenarioTests.cs

```diff
@@ -4402,9 +4402,9 @@ public void Test_Issue9467()
 output vaultId string = CertificateVault.id
 ";
 
-        var (parameters, _, _) = CompilationHelper.CompileParams(Services.WithFeatureOverrides(new(UserDefinedTypesEnabled: true)), ("parameters.bicepparam", bicepparamText), ("main.bicep", bicepTemplateText));
+        var (parameters, _, _) = CompilationHelper.CompileParams(("parameters.bicepparam", bicepparamText), ("main.bicep", bicepTemplateText));
```

Comment:
If it's not too involved, please could you include an evaluation test? Just to give confidence that the symbolic name codegen is actually going to be accepted and do the right thing as per the deployment engine: https://github.com/Azure/bicep/blob/main/src/Bicep.Core.IntegrationTests/EvaluationTests.cs

Reply:
I added a small evaluation test to make sure that `"languageVersion": "2.0"` was being set and that a feature that requires a language version (nullable types) was being evaluated correctly.